src/main/java/com/devchang202/calculator/Calculator.java

```diff
@@ -42,28 +42,28 @@ int getResultOfCalculator(String[] parsedEquation) {
         return result;
     }
 
-    int calculate(String op, int a, int b) {
+    int calculate(String op, int a, int b) throws IllegalArgumentException {
         switch (op) {
             case "+":
                 return add(a, b);
             case "-":
-                return substrate(a, b);
+                return subtract(a, b);
             case "*":
                 return multiply(a, b);
             case "/":
                 return divide(a, b);
             default:
-                throw new IllegalArgumentException();
+                throw new IllegalArgumentException("invalid operator");
         }
     }
 
-    void checkOperator(String op) {
+    void checkOperator(String op) throws  IllegalArgumentException {
```

Comment:
```suggestion
    void checkOperator(String op) throws IllegalArgumentException {
```

매의 눈 @_@
또한 위 예외처리에서만 메시지를 추가해주지 않고 모든 부분에 추가한다면 조금 더 친절한 예외메시지를 보실 수 있을 것 같습니다 :)